src/themes/shared/breakpoints.ts

```diff
@@ -1,9 +1,8 @@
-const breakpoints = {
+export const breakpoints = {
+  base: '0em',
```

Comment:
Sorry, why do we need to add this breakpoint. What could it be the case?

Reply:
It's related to the way chakra(emotion) builds the media queries based on the breakpoints. In the background, there will be something like `min-width: 1280px` https://chakra-ui.com/docs/styled-system/utility-hooks/use-media-query as a resulting underlying media query. By having a base in the theme we make sure to be able to target devices smaller than xs. It might be that I misunderstood something here... (Both the chakra base them and the fs one are providing the base breakpoint)